src/Data/CAS/RocksDB.hs

```diff
@@ -757,3 +770,79 @@ decIterKey it k = case B.splitAt (B.length namespace) k of
     namespace = _rocksDbTableIterNamespace it
 {-# INLINE decIterKey #-}
 
+data Checkpoint
+
+foreign import ccall unsafe "rocksdb\\c.h rocksdb_checkpoint_object_create" 
+  rocksdb_checkpoint_object_create :: C.RocksDBPtr -> Ptr CString -> IO (Ptr Checkpoint)
```

Comment:
Could you try to be consistent about indentation width. Here it's 2 but below it's 4 characters. Let's try to be consistent throughout the project.

Reply:
My editor is trying to infer it, and this started in a separate file, so it stuck :/ will fix.